hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -276,9 +276,19 @@ public static class Config implements Serializable {
             + ". Allows transforming raw source Dataset to a target Dataset (conforming to target schema) before "
             + "writing. Default : Not set. E:g - org.apache.hudi.utilities.transform.SqlQueryBasedTransformer (which "
             + "allows a SQL query templated to be passed as a transformation function). "
-            + "Pass a comma-separated list of subclass names to chain the transformations.")
+            + "Pass a comma-separated list of subclass names to chain the transformations. Transformer can also include "
```

Comment:
it's added to the start of the transformer (prefix) and end of property (suffix)?  lets call this an `id` instead of `suffix`? its easier to understand.

and if I understand correctly, there ordering of transformers should not matter. rename everywhere uniformly?

Reply:
Addressed in https://github.com/apache/hudi/pull/8514. Ordering is based on occurrence. Updated the docs for it. Currently its not ordered using the identifier only through position of transformer in the config.